### PR TITLE
Add a visualization for the filer

### PR DESCRIPTION
**Pull request by cschiano**
I added a small template to be able to browse the directories and the files instead of a raw json. Please do not hesitate to contact me if I can help.

Regards,
Clément

### CODE REVIEW

File: weed/server/filer_server_handlers_read.go [hunk 1]
```diff
@@ -42,63 +39,30 @@ func (fs *FilerServer) listDirectoryHandler(w http.ResponseWriter, r *http.Reque
 }
 
 func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request, isGetMethod bool) {
-	if strings.HasSuffix(r.URL.Path, "/") {
-		if fs.disableDirListing {
```
Comment (chrislusf) [inline weed/server/filer_server_handlers_read.go:46]: this option was skipped? Can we add this back and display it properly on the template?

File: weed/server/filer_server_handlers_read.go [hunk 2]
```diff
@@ -42,63 +39,30 @@ func (fs *FilerServer) listDirectoryHandler(w http.ResponseWriter, r *http.Reque
...
-			w.WriteHeader(http.StatusMethodNotAllowed)
-			return
-		}
-		fs.listDirectoryHandler(w, r)
-		return
-	}
+	fileLimit := 100
```
Comment (chrislusf) [inline weed/server/filer_server_handlers_read.go]: Can this fileLimit be an parameter?

File: weed/server/filer_server_handlers_read.go [hunk 3]
```diff
@@ -42,63 +39,30 @@ func (fs *FilerServer) listDirectoryHandler(w http.ResponseWriter, r *http.Reque
...
+	files, err := fs.filer.ListFiles(r.URL.Path, "", fileLimit)
```
Comment (chrislusf) [inline weed/server/filer_server_handlers_read.go]: The empty string here can be a filename, so that the pagination can work.

**Comment by chrislusf** [comment]
Thanks for this initiative! Please attach a screenshot since this is about UI changes.

Also, please try to add a basic pagination so that it can go to the next page if more than 100 files.

**Comment by cschiano** [comment]
Thank you for your feedback. I added back the check on the disableDirListing. The limit can be specified and a "load more" link is displayed if not all files have been loaded.
Please find attached some screenshots.
screenshot1
screenshot2
screenshot3

**Comment by chrislusf** [comment]
Thanks!

**Pull request merged**